uktrade/data-hub-frontend
# Global account managers filter for company page

## Description of change

Filters the new company list collection page by lead ita / global account manager.

## Test instructions

The new companies list page (/companies/react) should include a filter for lead ITA / Global Account Manager. The filter should work as a multi-select typeahead. Applying the filter should filter the results and update the url to include `one_list_group_global_account_manager[0]=<adviser_id>` A chip should appear above the list to show that the filter has been applied.

## Screenshots

![Screenshot from 2021-06-14 16-23-43](https://user-images.githubusercontent.com/1234577/121917401-fe155c80-cd2c-11eb-8eaa-050bfbb75460.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
